mtags/src/main/scala-3/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -51,14 +51,17 @@ object HoverProvider:
     val exprTp = typeFromPath(enclosing)
     val exprTpw = exprTp.widenTermRefExpr
 
-    if tp.isError || tpw == NoType || tpw.isError || path.isEmpty ||
-      (pos.isPoint // don't check isHoveringOnName for RangeHover
-        && source
-          .map(s => !MetalsInteractive.isOnName(enclosing, pos, s))
-          .getOrElse(false))
```

Comment:
Now, this check is done inside of `MetalsInteractive.enclosingSymbols`, it's not needed.